include/certifier_framework.h

```diff
@@ -29,7 +29,8 @@ typedef unsigned char byte;
 
 using std::string;
 
-// Policy store
+// -------------------------------------------------------------------
+// Policy store: Manage storage and administration of policies.
```

Comment:
@yelvmw -- We should walk thru each of this comment block and cleanup / improve them. I've tried to put down what seemed to be best suited.